code/go/0chain.net/blobbercore/handler/handler.go

```diff
@@ -451,13 +453,25 @@ func InsertShare(ctx context.Context, r *http.Request) (interface{}, error) {
 		return nil, common.NewError("share_info_insert", "Wrong ownerID or clientID")
 	}
 
+	var available common.Timestamp
+	if len(availableAfter) == 0 {
+		available = common.Now()
```

Comment:
Here might be the issue with timezones. We might better store 0 in the database, meaning it's always available.